examples/examples.ts

```diff
@@ -38,14 +38,26 @@ builder()
     .clusterProvider(new bp.FargateClusterProvider({...publicCluster}))
     .build(app, "fargate-blueprint");
 
-builder()
+const mng = builder()
     .clusterProvider(new bp.MngClusterProvider({
-        ...publicCluster
+        ...publicCluster, 
+
+        launchTemplate: {
+            requireImdsv2: true, 
+            tags: {
+                "cost-center": "2122",
```

Comment:
WHy is cost center required?

Reply:
not requited. it is an example of using LT tags

Reply:
ah ok. I think this should go to documentation too.

Reply:
we have docs on LT tags, but not example. I used to based on the defect brought up by the customer. I will eventually add docs for all examples in this repo. In general, we tend to use patterns repo for such.